src/main.rs

```diff
@@ -543,6 +543,28 @@ fn cmd_inspect(
         version: version.clone(),
     });
 
+    if sub_args.mode == FetchMode::Sourcegraph {
+        tokio::runtime::Handle::current()
+            .block_on(prompt_criteria_eulas(
+                out,
+                cfg,
+                network.as_ref(),
+                &store,
+                package,
+                None,
+                version,
+            ))
+            .into_diagnostic()?;
+
+        let url = format!("https://sourcegraph.com/crates/{package}@v{version}");
+        match open::that(&url) {
+            Ok(()) => return Ok(()),
```

Comment:
Let's log something to the console to let the user know we opened this page in their browser. In some systems, the user might be running such that the browser isn't brought to the front when they open it, and it could be quite surprising when the UI just doesn't show up because e.g. firefox stayed in the background, and the command just immediately exited.

I know I personally would be running in a situation where it might be weird, as I run commands on a remote SSH server, so any browser started will not be visible to me at all (as it's on a different computer)